jquery.fullPage.js

```diff
@@ -2076,22 +2080,33 @@
         function resizeHandler(){
             //checking if it needs to get responsive
             responsive();
+            if(document.body.classList.contains('fp-responsive')) return; // IVO GELOV

```

Comment:
When the **responsive** mode is active, we should not enforce the viewport height to the sections - their height should be left to AUTO.